README.rst

```diff
@@ -97,6 +97,10 @@ Please refer to below table for release support information:
      - Sagemaker TensorFlow Extensions Release Version
      - TensorFlow Release Version
      - Sagemaker TensorFlow Extentions Supported Python Versions
+   * - 2.9.0.1.15.x
+     - v1.15.x
+     - 2.9.0
```

Comment:
Tensorflow released 2.9.1 with a bug fix yesterday. Does this need to be updated?

Reply:
in that case we can release extension with 2.9.1 rather than 2.9.0

Reply:
updated to 2.9.1 due to bug fix